### PR TITLE
Add badge and initial content

### PR DESCRIPTION
**Pull request by jtoar**
This PR adds:
- The regular badge provided by the awesome repo (https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge)
- A suggested table of contents with initial content

Let me know if I should split up this PR / make more modular PRs, e.g. one per section. And if you have any feedback on the content: additions, removals--is it awesome enough?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,53 @@
...
+### Plugins
+- [nprogress](https://github.com/freddydumont/redwood-nprogress#readme)
+
+### Podcasts
+- [JS Party](https://changelog.com/jsparty/119)
+- [Fullstack Radio](http://www.fullstackradio.com/138)
+- [Shoptalk](https://shoptalkshow.com/412)
+
+### Projects Using Redwood
+- [Fizbuz](https://fizbuz.com/)
```
Comment (thedavidprice) [inline README.md:41]: Fizbuz changed up and is no longer trying to re-build on redwood. So strike from the list :( 

Maybe add Vida instead? 
https://community.redwoodjs.com/t/vida-turn-json-files-into-data-dashboard/462

**Comment by thedavidprice** [comment]
Rock! 🎸Thanks for getting this started. One comment about a change and addition -- take a look and we'll get this merged.

Once merged, will you create a new Topic in the forum announcing this? Would be great to come from you as a community-driven effort. Then I'll get the Twitter going...

File: README.md (hunk 1 above)
Comment (jtoar) [reply README.md]: Made that change.

**Comment by jtoar** [comment]
Made that change; and you bet!

**Pull request merged**